doc/articles/uno-publishing-windows.md

```diff
@@ -0,0 +1,99 @@
+---
+
+# Publishing Your App for Windows
+
+## Preparing For Publish
+
+## Building your app
+
+Building your app require using the msbuild command (`dotnet build` is not compatible as of WinAppSDK 1.5).
+
+### Side loading (without signing)
+
+#### Package the unsigned app
+
+Packaging the app without a code signature allows for installing the app on a machine without having to install the signer's certificate. This portion of the guide is derived from the [official Windows App SDK documentation](https://learn.microsoft.com/en-us/windows/msix/package/unsigned-package).
+
+To package your app:
+
+- Update the `Package.appxmanifest` file with the following `Identity` node to include the following `OID`:
+
+  ```xml
+  <Identity Name="MyApp"
+            Publisher="CN=AppModelSamples, OID.2.25.311729368913984317654407730594956997722=1"
+            Version="1.0.0.0" />
+  ```
+
+- Build your app in the folder of the app's `.csproj` by using the following command:
+
+    ```pwsh
+    msbuild /r /p:TargetFramework=net8.0-windows10.0.19041 /p:Configuration=Release /p:Platform=x64 /p:GenerateAppxPackageOnBuild=true /p:AppxBundle=Never /p:UapAppxPackageBuildMode=Sideloading /p:AppxPackageDir="C:/temp/output/" /p:AppxPackageSigningEnabled=false
+    ```
+
+> [!IMPORTANT]
+> Single package msix bundling is [not yet supported from msbuild the command line](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/single-project-msix?tabs=csharp#automate-building-and-packaging-your-single-project-msix-app).
+
+#### Install unsigned the app
+
+To install the app:
+
+- Start an elevated powershell command prompt (Search for powershell in the start menu, right click on it then Run as administrator):
+- In the folder contaning the `.msix` file, execute the following command :
+
+    ```pwsh
+    Add-AppPackage -AllowUnsigned ".\MyApp.appx"
+    ```
+
+For more information, see the [official documentation](https://learn.microsoft.com/en-us/windows/msix/package/unsigned-package#install-an-unsigned-package).
+
+### Side loading (with signing)
+
+#### Package signed the app
+
+This guide uses a self-signed certificate.
+
+To package your app:
+
+- Create a self-signed certificate:
+  - Open the solution in Visual Studio 2022
+  - Double click on the `Package.appxmanifest` file
+  - Navigate to the `Packaging` tab
+  - Click the **Create** button, then set a **publisher common name**, then OK. Do not se  password.
+  - Close the **Choose a Certificate** window by clicking OK.
+  - Click the **Save file** button in the Visual Studio toolbar
+- Build the app on the command line in the folder of the app's `.csproj`, use the following command:
+
+    ```cmd
+    msbuild /r /p:TargetFramework=net8.0-windows10.0.19041 /p:Configuration=Release /p:Platform=x64 /p:GenerateAppxPackageOnBuild=true /p:AppxBundlePlatforms="x86|x64|arm64" /p:AppxBundle=Never /p:UapAppxPackageBuildMode=Sideloading /p:AppxPackageDir="C:\temp\output\\" /p:AppxPackageSigningEnabled=true
```

Comment:
```suggestion
### Side loading (with signing)

#### Package signed the app

This guide uses a self-signed certificate.

To package your app:

- Create a self-signed certificate:
  - Open the solution in Visual Studio 2022
  - Double-click on the `Package.appxmanifest` file
  - Navigate to the `Packaging` tab
  - Click the **Create** button, then set a **publisher common name**, then OK. Do not set a password.
  - Close the **Choose a Certificate** window by clicking OK.
  - Click the **Save file** button in the Visual Studio toolbar
- Build the app on the command line in the folder of the app's `.csproj`, use the following command:

```cmd
msbuild /r /p:TargetFramework=net8.0-windows10.0.19041 /p:Configuration=Release /p:Platform=x64 /p:GenerateAppxPackageOnBuild=true /p:AppxBundlePlatforms="x86|x64|arm64" /p:AppxBundle=Never /p:UapAppxPackageBuildMode=Sideloading /p:AppxPackageDir="C:\temp\output\\" /p:AppxPackageSigningEnabled=true
```

Reply:
Also @jeromelaban, where is the "Create" button you are mentioning?
![image](https://github.com/unoplatform/uno/assets/16295702/bdad8011-755b-4bde-8482-32ba78895052)

Reply:
Ah it's missing the Choose Certificate button step :) Will add.